server/resource_server/emails/verification/code/generator.go

```diff
@@ -0,0 +1,5 @@
+package code
+
+type Generator interface {
```

Comment:
move this into the same folder as the struct definition to improve code readability

Reply:
Probably it doesn't make sense to do it now, because in the next PR I'll add another interface implementation